docs/examples/fastify-webhook-bot.js

```diff
@@ -22,6 +22,4 @@ bot.telegram.setWebhook(WEBHOOK_URL).then(() => {
   console.log('Webhook is set on', WEBHOOK_URL)
 })
 
-app.listen(PORT, () => {
-  console.log('Start listening on port', PORT)
-})
+app.listen(PORT).then(console.log)
```

Comment:
Printing bare address would be confusing, how about

```suggestion
app.listen(PORT).then(() => {
  console.log('Start listening on port', PORT)
})
```
or even just
```suggestion
app.listen(PORT)
```